docs/tutorial.rst

```diff
@@ -315,10 +315,16 @@ The determination between the two modes is done well ahead of any execution.
 You do not need to worry about partially executed commands - that is
 impossible.
 
-If you absolutely want to run a subprocess command, you can always
-force xonsh to do so with the syntax that we will see in the following
-sections.
+If you still have concerns about the determination process and want to have strong confidence
+when running a subprocess command in your xonsh scripts, you can always use the elegant subprocess
+mode syntax that we will see in the following sections. Short example now:
```

Comment:
Maybe
```suggestion
If you know that you want to run a command in subprocess mode, you can force subprocess execution by using the following syntax:
```

Reply:
Thanks! I've chosen the scopatz version. It has more strict text.